src/Microsoft.Graph/Generated/callrecords/requests/CallRecordRequest.cs

```diff
@@ -60,6 +60,29 @@ public async System.Threading.Tasks.Task<CallRecord> CreateAsync(CallRecord call
             return newEntity;
         }
 
+        /// <summary>
+        /// Creates the specified CallRecord using POST and returns a <see cref="GraphResponse{CallRecord}"/> object.
+        /// </summary>
+        /// <param name="callRecordToCreate">The CallRecord to create.</param>
+        /// <returns>The <see cref="GraphResponse{CallRecord}"/> object of the request.</returns>
+        public System.Threading.Tasks.Task<GraphResponse<CallRecord>> CreateResponseAsync(CallRecord callRecordToCreate)
+        {
+            return this.CreateResponseAsync(callRecordToCreate, CancellationToken.None);
+        }
+
+        /// <summary>
+        /// Creates the specified CallRecord using POST and returns a <see cref="GraphResponse{CallRecord}"/> object.
+        /// </summary>
+        /// <param name="callRecordToCreate">The CallRecord to create.</param>
+        /// <param name="cancellationToken">The <see cref="CancellationToken"/> for the request.</param>
+        /// <returns>The <see cref="GraphResponse{CallRecord}"/> object of the request.</returns>
+        public async System.Threading.Tasks.Task<GraphResponse<CallRecord>> CreateResponseAsync(CallRecord callRecordToCreate, CancellationToken cancellationToken)
+        {
+            this.ContentType = "application/json";
```

Comment:
could we imagine refer to constants instead of redefining all these values multiple times?